digitalocean/datalist/schema.go

```diff
@@ -0,0 +1,75 @@
+package datalist
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type DataListResourceConfig struct {
```

Comment:
Exported types and functions should have comments. That's generally good practice, but I think it's especially important in this case. We want anyone familiar with Terraform providers to be able to pick up this code base quickly. As this is something custom to this provider, documenting it is important.

Also, since this will be called as `datalist.DataListResourceConfig`, we might want to consider renaming it `ResourceConfig` so it is less repetitive.

Reply:
Added comments and renamed the config struct. Let me know if you would like me to fill in more details.